src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -134,6 +134,11 @@ public async Task<bool> DownloadQualityProfileAsync(IProgress<FixedStepsProgress
             {
                 var serverLanguage = language.ServerLanguage;
 
+                if (serverLanguage == null)
```

Comment:
@rita-sonarsource this looks like your commit. Should it be there or is it some merge mistake and should be removed?

Reply:
Merge mistake, should be removed